tests/test_forms.py

```diff
@@ -25,3 +28,39 @@ def test_only_ons_email_address_valid(self):
                 result = form.validate()
                 self.assertTrue(result)
                 self.assertEqual(len(form.errors), 0)
+
+    def test_valid_password_for_activation_and_reset(self):
+        with self.app.app_context():
+            form = SetAccountPasswordForm()
+            form.password.data = "Abcde123Edcba!"
+            form.password_confirm.data = "Abcde123Edcba!"
+            result = form.validate()
+            self.assertTrue(result)
+            self.assertTrue(len(form.errors) == 0)
+
+    def test_nonvalid_password_for_activation_and_reset(self):
```

Comment:
nonvalid or invalid? and same as above regarding name (test_set_account_password_invalid)

Reply:
Updated.